sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
-## 2.1.0-beta.1 (Unreleased)
+## 2.1.0 (2023-10-23)
```

Comment:
If we're not releasing any new public APIs we should make this a patch update instead.
```suggestion
## 2.0.1 (2023-10-23)
```